main.py

```diff
@@ -1,6 +1,16 @@
 from retrieveData import readSqliteTable
 from databaseInsert import insertIntoDatabase
+from create_db import createDb
+import os.path
+import json
+import sys
 
 # Main function to call the functions as needed by the user.
 if __name__ == "__main__":
+  if (not os.path.exists("Recipes.db")):
+    createDb()
+  with open(sys.argv[1]) as f:
```

Comment:
Consider using argparse for command line arguments

Reply:
Made a ticket for this.